rest/bulk_api.go

```diff
@@ -295,8 +285,11 @@ func (h *handler) handleDumpChannel() error {
 // 	 ]
 // }
 func (h *handler) handleBulkGet() error {
-	includeRevs := h.getBoolQuery("revs")
 	includeAttachments := h.getBoolQuery("attachments")
+	revsLimit := 0
+	if h.getBoolQuery("revs") {
+		revsLimit = int(h.getIntQuery("revs_limit", math.MaxInt32))
```

Comment:
The problem is that the default for `revs_limit` is "infinity", i.e. a huge value, but there's no `MaxInt` constant, as far as I know. So I had to choose `MaxInt32` because `MaxInt64` would break if assigned to an `int` in a 32-bit build.

The reason for having to cast back to `int` is that `getIntQuery` returns an `int64`, because there are other places that need to get 64-bit values from URL query params.